Projects/UOContent/Items/Armor/BaseArmor.cs

```diff
@@ -1451,6 +1452,12 @@ public override void GetProperties(IPropertyList list)
 
         public override void OnSingleClick(Mobile from)
         {
+            if (!Core.AOS)
```

Comment:
Per above, this would be Core.T2A. I have to think about the logic. We might want the single click request to go to OnSingleClickT2A or OnSingClickAOS or OnSingleClick depending on era. Then OnSingleClickT2A would execute OnSingleClick by default unless it's overridden.